lib/kits/paged-publication/event-tracking.ts

```diff
@@ -36,11 +36,11 @@ class PagedPublicationEventTracking extends MicroEvent {
         return this;
     }
 
-    trackPageSpreadDisappeared(pageNumbers: number[]) {
+    trackPageSpreadAppeared(pageNumbers: number[]) {
```

Comment:
Could also just call it trackPageSpread and alias trackPageSpreadDisappeared to trackPageSpread.